src/Skia/Avalonia.Skia/WriteableBitmapImpl.cs

```diff
@@ -15,9 +15,11 @@ namespace Avalonia.Skia
     /// </summary>
     internal class WriteableBitmapImpl : IWriteableBitmapImpl, IDrawableBitmapImpl
     {
-        private static readonly SKBitmapReleaseDelegate s_releaseDelegate = ReleaseProc;
```

Comment:
It was here for a reason. You have basically disabled our leak detection system.

Reply:
The leak detection is still in place. If the blob's finalizer is run without disposal it still generates a message.